test/tests/functional/pbs_qstat.py

```diff
@@ -57,6 +58,75 @@ def parse_json(self, dictitems, qstat_attr):
                         self.parse_json(val, qstat_attr)
         return qstat_attr
 
+    def validate_el(self, attr_dict, user):
+        """
+        Check if the value of server attributes reported by
+        qstat -Bf matches the expected values.
+        """
+        qstat_o = {}
+        qstat_o['server_state'] = 'Active'
+        qstat_o['server_host'] = self.server.hostname
+        qstat_o['scheduling'] = 'True'
+        qstat_o['total_jobs'] = '0'
+        qstat_o['state_count'] = ['Transit', 'Queued',
+                                  'Held', 'Waiting', 'Running',
+                                  'Exiting', 'Begun']
+        qstat_o['default_queue'] = 'workq'
+        qstat_o['log_events'] = '511'
+        qstat_o['mail_from'] = 'adm'
+        qstat_o['query_other_jobs'] = 'True'
+        qstat_o['resources_default.ncpus'] = '1'
+        qstat_o['default_chunk.ncpus'] = '1'
+        qstat_o['resv_enable'] = 'True'
+        qstat_o['node_fail_requeue'] = '310'
+        qstat_o['max_array_size'] = '10000'
+        qstat_o['pbs_license_linger_time'] = '3600'
+        qstat_o['pbs_version'] = self.server.pbs_version
+        qstat_o['eligible_time_enable'] = 'False'
+        qstat_o['max_concurrent_provision'] = '5'
+        qstat_o['FLicenses'] = None
+        qstat_o['license_count'] = None
+        qstat_o['managers'] = None
+        qstat_o['flatuid'] = None
+        qstat_o['id'] = self.server.shortname
+
+        # If test case is run as root, PTL sets acl_roots
+        if os.getuid() == 0:
```

Comment:
Just a suggestion take it leave it up to you: You can merge this if condition with below if condition

Reply:
But there is a difference. os.getuid() is in the case when the test case is being run as root. The other if condition is to check if qstat -Bf is executed as root.

Reply:
Ah you are right! you need separate if condition